### PR TITLE
Adjust phpunit configuration

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] references `phpunit.xsd` using a relative path
* [x] adjusts configuration to be as strict as possible
* [x] increases verbosity

Follows https://github.com/php-coveralls/php-coveralls/pull/242#discussion_r136697081.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -13,6 +18,7 @@
     convertWarningsToExceptions="true"
     processIsolation="false"
     stopOnFailure="false"
+    verbose="true"
```
Comment (keradus) [inline phpunit.xml.dist]: with this, do we still need to add `-v` when we run tool under Travis/AppVeyor/CircleCI ?

Comment (localheinz) [reply phpunit.xml.dist]: I don't think so, let me remove it!

**Comment by keradus** [comment]
Thank you @localheinz.

**Comment by keradus** [comment]
@localheinz 
now, we have 150 risky tests on 2.2 line like:
```
150) PhpCsFixer\Tests\Linter\TokenizerLinterTest::testLintSource with data set #0 ('<?php echo 123;')
This test did not perform any assertions
```

could you fix it please ?

**Comment by localheinz** [comment]
Thank you, @julienfalque, @keradus, and @SpacePossum!

Will take a look st the risky tests tomorrow!

**Pull request closed without merging**